gpii/node_modules/flowManager/src/FlowManager.js

```diff
@@ -180,7 +182,20 @@ fluid.defaults("gpii.flowManager.local", {
             type: "gpii.settingsHandlers.webSockets.component"
         },
         pspChannel: {
-            type: "gpii.pspChannel"
+            type: "gpii.pspChannel",
+            options: {
+                members: {
+                    // This member value is calculated here rather than in the pspChannel component itself is to ensure
```

Comment:
It's not necessary to write them here - the framework doesn't consider the site where options are written in deciding on expansion order, which is always data-driven. However, given that the options do indeed reach out into several sibling components, it's not unreasonable to express them at a higher integration level. But yet on the other hand - most of the options for the pspChannel already reach out into lifecycleManager and flowManager so I suggest after all just putting them down there with the others.